bigtable/tools/convert_tests.py

```diff
@@ -0,0 +1,82 @@
+# Copyright 2017 Google Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+# This script generates C++ unit tests from the json specification for
+# acceptance tests found in read-rows-acceptance-test.json.
+#
+# Usage:
+#   python convert_tests.py <read-rows-acceptance-test.json >rr.cc
```

Comment:
Consider:

```console
$ python convert_tests.py <read-rows-acceptance-test.json >generated-read-rows-tests.inc
```

and `#include` that file from the test.  You can also get cmake to download the file from whatever is the source for it.

Reply:
Also if you decide to commit the generated code to the repo, consider:

```console
$ python convert_tests.py <read-rows-acceptance-test.json | \
        clang-format >generated-read-rows-tests.cc
```